packages/playground-examples/copy/pt/JavaScript/JavaScript Essentials/Code Flow.ts

```diff
@@ -0,0 +1,23 @@
+//// { title: "Fluxo de código", order: 3, compiler: { strictNullChecks: true } }
+
+// Como o código flui dentro de nossos arquivos JavaScript pode afetar
+// os tipos em nossos programas.
+
+const users = [{ name: 'Ahmed' }, { name: 'Gemma' }, { name: 'João' }]
+
+// Vamos ver se conseguimos encontrar um usuário chamado "João".
+const joao = users.find(u => u.name === 'João')
```

Comment:
I switched the code from "Jon" to "João" to fit the audience. I think _some_ of this is good, but I would ask that you please keep a diverse set of names in the examples (gender / nationality is mainly what I strive for)